### ISSUE TITLE
[Image normalization before feeding to the backbone model]

### ISSUE THREAD

**Issue by shubhamkriitr** [niamhbelton/FewSOME#3]
Hi, in the following snippet the image (x) is being passed as is. However, the base model would expect normalized image. The image was not normalized elsewhere in the code. Is it supposed to be like this? 
```py

class RESNET_pre(nn.Module):
  def __init__(self):
      super(RESNET_pre, self).__init__()

      self.features = models.resnet18(pretrained=True)


  def forward(self, x):
      x = torch.unsqueeze(x, dim =0)
      x= self.features(x)
      x=nn.Sigmoid()(x)

      return x #output
```
